lib/orderbook/OrderBook.ts

```diff
@@ -98,25 +105,41 @@ class OrderBook extends EventEmitter {
     return this.addOwnOrder(order);
   }
 
-  public addMarketOrder = (order: orders.MarketOrder) => {
+  public addMarketOrder = (order: orders.OwnMarketOrder): matchingEngine.MatchingResult => {
     const price = order.quantity > 0 ? Number.MAX_VALUE : 0;
     return this.addOwnOrder({ ...order, price }, true);
   }
 
-  public removeOwnOrder = (orderId: string, pairId: string): boolean => {
+  public removeOwnOrderByLocalId = (pairId: string, localId: string): boolean => {
+    const id = this.localIdMap[localId];
+
+    if (id === undefined) {
+      return false;
+    } else {
+      delete this.localIdMap[localId];
```

Comment:
It would be better to have the API balanced, so if we're adding to `localIdMap` at `addOrder`, we should remove it from there at `removeOrder`, and not 2 levels above that. But it's fine to leave it like this for now, because of the upcoming #215.